app/controllers/notification_settings_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
     @notification_settings = []
 
     notifications.each do |notification|
-      @notification_settings.push NotificationSetting.find_or_initialize_by(notification_type: notification)
+      @notification_settings.push NotificationSetting.find_or_initialize_by(notification_type: notification, user_id: current_user.id)
```

Comment:
this fix is good for sure

Reply:
would be great to add a test for this in rad_common in notification_settings_controller_spec

Reply:
actually, since we are setting that user in the create, is that needed? we could put the user attribute in the form as hidden if that helps, then we wouldn't have to set it to current_user in the create